ui/src/errors.ts

```diff
@@ -6,9 +6,17 @@
 import { localize } from "./localize";
 
 export class UserCancelledError extends Error {
+    private readonly _isUserCancelledError: boolean = true;
+
     constructor() {
         super(localize('userCancelledError', 'Operation cancelled.'));
     }
+
+    // Workaround to fix "instanceof UserCancelledError" when testing package with "npm link"
+    // tslint:disable-next-line:function-name no-any
+    public static [Symbol.hasInstance](instance: any): boolean {
```

Comment:
Can you explain what this is?? I'm guessing it's overloading, but just curious as to how.

Reply:
Just overwrites default behavior of "instanceof"
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/hasInstance

> The Symbol.hasInstance well-known symbol is used to determine if a constructor object recognizes an object as its instance. The instanceof operator's behavior can be customized by this symbol.